tests/jballerina-unit-test/src/test/resources/test-src/query/simple-query-with-defined-type.bal

```diff
@@ -151,16 +174,16 @@ function testFilteringNullElements() returns Person[] {
     return outputPersonList;
 }
 
-function testMapWithArity () returns string[] {
-    map<any> m = {a:"1A", b:"2B", c:"3C", d:"4D"};
+function testMapWithArity() returns string[] {
+    map<any> m = {a: "1A", b: "2B", c: "3C", d: "4D"};
     string[] val = from var v in m
                    where <string> v == "1A"
```

Comment:
What will happen if we remove the cast <string> ?

Reply:
it will throw error since map is any type map. Isn't that expected?

Reply:
It is a compile time error. (i.e, operator '==' not defined for 'any' and 'string')

Reply:
sorry, just now saw. That is an expected case.

Reply:
ah yes.. sorry didnt see that m is a map<any>